fsharp/oxpecker/Program.fs

```diff
@@ -0,0 +1,34 @@
+open Microsoft.AspNetCore.Builder
+open Microsoft.AspNetCore.Hosting
+open Microsoft.Extensions.DependencyInjection
+open Microsoft.Extensions.Hosting
+open Microsoft.Extensions.Logging
+open Oxpecker
+
+let userNameHandler (name: string) : EndpointHandler = _.WriteText(name)
+
+let endpoints =
+    [ GET [ route "/" <| text ""; routef "/user/{%s}" text ]
```

Comment:
For maximum performance `routef` should be substituted to manual route value retrieval ([example](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/FSharp/oxpecker/src/App/Program.fs#L58) )

Reply:
Also, for max perf, first GET handler could just return `Task.CompletedTask` or write content type header if it's required ([example](https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/FSharp/oxpecker/src/App/Program.fs#L84))